modules/ROOT/pages/plugins/opensource/help.adoc

```diff
@@ -59,7 +60,7 @@ tinymce.activeEditor.plugins.help.addTab({
 
 == Exposing metadata for the help plugin
 
-For information on how to expose metadata from you custom plugin to add it to the Installed plugins list in the Help plugin, see the  link:{baseurl}/advanced/creating-a-plugin/#exposingmetadata[Creating a Plugin page].
+For information on how to expose metadata from you custom plugin to add it to the Installed plugins list in the Help plugin, see the  xref:advanced/creating-a-plugin.adoc[Creating a Plugin page].
```

Comment:
This probably should link to the `Registering a custom plugin with {productname}` title instead of just linking to the root of the page, as based on the old `exposingmetadata` anchor that appears where it used to point to before the page got reworked.

Reply:
Also the text of the link says `Creating a Plugin page` so linking to the page itself doesn't seem too far off I think. Should this then be resolved ?

Reply:
@lnewson ☝️

Reply:
This really isn't a question for me to answer sorry. I was hoping others might answer... anyways for now lets just leave it as is.